packages/gatsby/src/utils/webpack-utils.ts

```diff
@@ -784,7 +784,7 @@ export const createWebpackUtils = (
       ],
       ...eslintConfig(schema, jsxRuntimeExists),
     }
-    //@ts-ignore
+    // @ts-ignore
```

Comment:
Now I'm wondering why our ts rules are allowing this!

Reply:
Do we have a lint rule for the space in typescript?

Reply:
No, but we do have a rule that requires a description when there's a ts-ignore comment

Reply:
It's only a warning :)
```
        "@typescript-eslint/ban-ts-comment": [
          `warn`,
          { "ts-ignore": `allow-with-description` },
        ],
```